### DISCUSSION TITLE
3x6 unicol fusion file

### DISCUSSION THREAD

**Discussion by senseored** [juhakaup/keyboards#24] [General]
Hi,
I really love the project (I've made two versions for myself - a 3x6 and a 3x6 unicol). I prefer the unicol, but would like to make some modifications for myself. Is there any possibility of you adding the 3x6 unicol Fusion 360 file to the project?

Thanks!

**Comment by juhakaup** [comment]
I added the file to the 'requests and other experiments' -folder.

**Reply by senseored** [reply]
Thank you, I really appreciate it.
